iotdb/src/main/java/org/apache/iotdb/db/service/JDBCServiceEventHandler.java

```diff
@@ -52,8 +56,7 @@ public void deleteContext(ServerContext arg0, TProtocol arg1, TProtocol arg2) {
 
   @Override
   public void preServe() {
-    // TODO Auto-generated method stub
-
```

Comment:
super.preServe() first?

Reply:
![image](https://user-images.githubusercontent.com/8021527/53059173-f3e24b00-34f0-11e9-8ee5-1b30f489b6fc.png)

Reply:
Above is thrift source code.
When server starts, it will first execute preServe() function, then receive client requests.